configs/mmflow/raft/opticalflow_onnxruntime_static.py

```diff
@@ -0,0 +1,5 @@
+_base_ = [
```

Comment:
structure of model config is `configs/{codebase}/{deploy_config_filename}`, pls .change it

Reply:
rename file from `configs/mmflow/raft/opticalflow_onnxruntime_static.py` to `configs/mmflow/opticalflow_onnxruntime_static.py`